### PR TITLE
have the linux build use $release_bin wait-ci

### PR DESCRIPTION
**Pull request by oconnor663**
r? @cjb @gabriel 

Please take a look in particular at whether I've done the right thing with the `--context` flags. I copied what was in the OSX build (minus the updater tests), but this stuff sounds like it's been changing recently?

### CODE REVIEW

File: packaging/linux/build_and_push_packages.sh [hunk 1]
```diff
@@ -22,13 +23,19 @@ echo "Using BUCKET_NAME $BUCKET_NAME"
...
-  rm -r "$temp"
-)
+echo "Loading release tool"
+release_gopath="$HOME/release_gopath"
+GOPATH="$release_gopath" "$client_dir/packaging/goinstall.sh" "github.com/keybase/release"
+release_bin="$release_gopath/bin/release"
+
+if [ ! "${NOWAIT:-}" = "1" ]; then
+  echo "Checking client CI"
+  "$release_bin" wait-ci --repo="client" --commit="$(git -C $client_dir rev-parse HEAD)" --context="continuous-integration/travis-ci/push" --context="continuous-integration/appveyor/branch" --context="ci/circleci"
```
Comment (oconnor663) [inline packaging/linux/build_and_push_packages.sh]: @cjb have we stopped using AppVeyor entirely?

**Comment by oconnor663** [comment]
Will need to pull from https://github.com/keybase/client/pull/3010 when that's done.

**Comment by cjb** [comment]
@oconnor663 ready for you now in #3010

**Comment by oconnor663** [comment]
Ok, updated to match the changes in #3010.

**Comment by gabriel** [comment]
it might be worth including the updater ci check just so we don't have to add it later if we use the new updater on linux?

**Comment by gabriel** [comment]
otherwise lg2m

**Comment by oconnor663** [comment]
We don't currently clone the updater at all as part of the Linux build. I'd prefer not to add that clone until we actually use the code, if that's ok.

**Pull request merged**